command/pipelines/pipeline_save.go

```diff
@@ -49,11 +51,23 @@ func (c *PipelineSaveCommand) flagSet() *flag.FlagSet {
 
 // parsePipelineFile reads and deserializes the input pipeline file.
```

Comment:
is this comment out of date now that the func falls back to stdin?  i guess stdin is technically a file.

Reply:
Yep.

Reply:
Yep as in it's out of date and I updated the comment*